### ISSUE TITLE
Improve visual look of the highlight 

### ISSUE THREAD

**Issue by rehno-lindeque** [opensourceBIM/BIMsurfer-before2019#36]
Improve the visual look of the highlight with a colortrans (color transformation) node: https://groups.google.com/forum/?hl=en#!topic/scenejs/2RMt-TKVbPg

Or alternatively used proposed change to materials: https://github.com/xeolabs/scenejs/issues/131

**Comment by xeolabs** [comment]
For now, you could try a nice yellow like in the Human:

``` javascript
SceneJS.scene("my-scene").findNode("my-colortrans").set({

            /* Saturation [-1.0 - 1.0] - negative value to desaturate
             */
            saturation: 0.0,

            /* Color addition/subtraction - negative value to subtract
             */
            add :{
                r: 0.35,
                g: 0.35,
                b: 0.0,
                a: 0.0
            },

            /* Color scaling
             */
            scale: {
                r: 1.0,
                g: 1.0,
                b: 1.0,
                a: 1.0
            }
});
```

**Comment by rehno-lindeque** [comment]
Thanks :D Do you think I have too much blue in the app? I still need work on my palette a bit ;)

**Comment by xeolabs** [comment]
Just a note that I'm about to remove the colortrans node and replace it with a custom shader - the effect will be the same, and you can always rewrite that shader to suit.

**Comment by rehno-lindeque** [comment]
Thanks, so you're saying I should just create my own custom "highlight" shader right?

**Comment by xeolabs** [comment]
I've just created a custom shader example that replicates the exact
effect of the colortrans node (cut and pasted from that), so you
should be able to use that..I'm just about to switch Human over to use
it as well.

On Sun, Oct 9, 2011 at 7:24 PM, Rehno Lindeque
reply@reply.github.com
wrote:

**Comment by rehno-lindeque** [comment]
Cool, that's great

**Comment by rehno-lindeque** [comment]
Also need to highlight objects that are composed of many objects (should be a similar fix to issue #32)
